spring-cloud-consul-bus/src/main/java/org/springframework/cloud/consul/bus/ConsulBusAutoConfiguration.java

```diff
@@ -68,16 +69,17 @@ public ConsulOutboundEndpoint consulOutboundEndpoint() {
 
 	@Bean
 	public IntegrationFlow cloudBusConsulOutboundFlow(
-			@Qualifier("cloudBusOutboundChannel") MessageChannel cloudBusOutboundChannel) {
+			@Qualifier("cloudBusOutboundChannel") MessageChannel cloudBusOutboundChannel,
```

Comment:
As I mentioned in #119 
> in multiple configuration classes injection of dependencies is not really injecting them, but creating a separate copy (because it's attempted via a method call on the same class)

so i fixed all the occurrences I was able to find